src/ray/raylet/worker.h

```diff
@@ -88,11 +95,20 @@ class Worker {
   // of a task.
   ResourceIdSet task_resource_ids_;
   std::unordered_set<TaskID> blocked_task_ids_;
+  /// How many heartbeats have been missed for this worker.
+  int num_missed_heartbeats_;
+  /// Indicate we have sent kill signal to the worker if it's true. We cannot treat the
+  /// worker process as really dead until we lost the heartbeats from the worker.
+  bool is_being_killed_;
   /// The `ClientCallManager` object that is shared by `WorkerTaskClient` from all
   /// workers.
   rpc::ClientCallManager &client_call_manager_;
   /// The rpc client to send tasks to this worker.
   std::unique_ptr<rpc::WorkerTaskClient> rpc_client_;
+  /// Reply of the `GetTask` request.
```

Comment:
Can you write in this comment why we have to store the reply and callback for `GetTask` requests?

Reply:
It seems Hao Chen has added a detailed comment in SetGetTaskReplyAndCallback definition to explain why we have to store them.